node_modules/gh-orgunit/lib/api.js

```diff
@@ -269,6 +269,11 @@ var _augmentOrgUnits = function(ctx, orgUnits, groups) {
             // Determine whether the user is subscribed to the serie
             augmentedSerie.subscribed = !_.isEmpty(serie.Calendars);
 
+            // Determine whether the user can manage the serie
+            var canManageSerieThroughGroups = !_.isUndefined(_.find(groups, {'id': serie.GroupId}));
+            var canAdminSerie = (ctx.user && ctx.user.canAdmin(serie.AppId));
+            augmentedSerie.canManage = canManageSerieThroughGroups || canAdminSerie || false;
```

Comment:
No, as canAdminSerie can be `null` (in case of anonymous). We do something similar a couple of lines higher